components/designSystem/SmolAddressInput.tsx

```diff
@@ -20,11 +19,12 @@ import type {TAddress, TInputAddressLike} from '@utils/tools.address';
 type TAddressInput = {
 	onSetValue: (value: TInputAddressLike) => void;
 	value: TInputAddressLike;
+	initialStateFromUrl?: string | undefined;
 };
 
-export function SmolAddressInput({onSetValue, value}: TAddressInput): ReactElement {
+export function SmolAddressInput({onSetValue, value, initialStateFromUrl}: TAddressInput): ReactElement {
 	const {onOpenCurtain, getEntry, getCachedEntry} = useAddressBook();
-	const {initialStateFromUrl} = useSendFlow();
```

Comment:
Found a bad practice: we shouldn't rely on any context inside of the reusable components